src/components/Card/Card.css

```diff
@@ -0,0 +1,9 @@
+.card {
+    height: 200px;
+    width: 200px;
```

Comment:
it's tough to hardcode this value without knowing the width of the parent (List). try using something like `width: 100%`